lib/ledger_sync/adaptors/quickbooks_online/adaptor.rb

```diff
@@ -80,6 +81,7 @@ def refresh!
           @access_token = refreshed.token
 
           @expires_at = Time&.at(refreshed.expires_at.to_i)&.to_datetime
+          @refresh_token_expires_at = Time&.at(Time.now.to_i + refreshed.params['x_refresh_token_expires_in'])&.to_datetime
```

Comment:
You should make this nilsafe if `refreshed.params['x_refresh_token_expires_in']` is `nil`

Reply:
Done, but it's in the spec so it really shouldn't be nil or we probably have other problems.